FoodOrderingSystem.py

```diff
@@ -0,0 +1,884 @@
+                cust_name = input("Enter customer name: ")
+                cust_phone = input("Enter customer phone number: ")
+                cust_email = input("Enter customer email address: ")
+                fos.customer_signup(cust_name, cust_phone, cust_email)
+
+            elif customer_options == 3:
+                cust_id = input("Enter customer ID: ")
+                fos.customer_login(cust_id)
+
+                selection = """ 
+
+                0. Logout
+                1. Process order
+                2. View order
+                3. Checkout
+                4. Cancel order
+                5. View order status
+                
+                Select option: 
+
+                """
+                order = int(input(selection))
+
+                while order != 0:
+
+                    if order == 1:
+                        fos.process_order(cust_id)
+
+                    elif order == 2:
+                        order_id = input("Enter order ID: ")
+                        fos.view_order(order_id)
+                    # Problem - A customer can look at any order using the order ID. Need to relook how to solve this.
+                
+                    elif order == 3:
+                        order_id = input("Enter order ID: ")
+                        order_address = input("Enter delivery address: ")
+                        checkout = int(input("Press 1 to confirm checkout: "))
+                        if checkout == 1:
+                            order_status = "Checkedout"
+                        checkout_time = datetime.now()
+                        delivery_time = timedelta(minutes=30)
+                        estimated_time = checkout_time + delivery_time
+                        session = fos.Session()
+                        view_grand_total = fos.customer.view_order_grand_total(order_id, session)
+                        if view_grand_total:
+                            for fd, cos, cd, cosa, t in view_grand_total:
+                                bill_amount = t
+                        fos.checkout(order_id, order_status, order_address, checkout_time, estimated_time, bill_amount)
+                    # Problem - A customer can look at any order using the order ID. Need to relook how to solve this.
+
+                    elif order == 4:
+                        order_id = input("Enter order ID: ")
+                        cancel = int(input("Press 1 to confirm cancellation: "))
+                        if cancel == 1:
+                            order_status = "Cancelled"
+                        fos.cancel_order(order_id, order_status) 
+                    # Problem - A customer can look at any order using the order ID. Need to relook how to solve this.
+
+                    elif order == 5:
+                        order_id = input("Enter order ID: ")
+                        fos.view_orders_status(order_id)
+
+                    order = int(input("\nSelect option: "))               
```

Comment:
Put this into another function `process_customer_options_flow()` and call it from main()

Reply:
Changes made.